pkgs/tools/security/nwipe/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchFromGitHub, ncurses, parted, automake, autoconf, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  version = "0.24";
+  name = "nwipe-${version}";
+  src = fetchFromGitHub {
+    owner = "martijnvanbrummelen";
+    repo = "nwipe";
+    rev = "v${version}";
+    sha256 = "0zminjngz98b4jl1ii6ssa7pkmf4xw6mmk8apxz3xr68cps12ls0";
+  };
+  nativeBuildInputs = [ automake autoconf ncurses parted pkgconfig ];
```

Comment:
`parted` and `ncurses` should be `buildInputs`, not `nativeBuildInputs` (they don't need to run during the build).